### ISSUE TITLE
excerpt_length length is ignored when called with string

### ISSUE THREAD

**Issue by gnesher** [cabrerahector/wordpress-popular-posts#115]
When using the following form:
`
<?php
wpp_get_mostpopular( 'header="text"&stats_views=0&limit=5&range="monthly"&post_type="post"&excerpt_length=5&post_html="<li>{title} <div class=\'latest-text\'>{summary}</div></li>"' );
?>
`
The excerpt_length is being ignored (other values seems to work).

**Comment by cabrerahector** [comment]
Hi there!

I tried your code in my site's footer.php using both styles (string vs array):

```php
<?php
if ( function_exists('wpp_get_mostpopular') ) {
	
	// string version
	wpp_get_mostpopular( 'header="text"&stats_views=0&limit=5&range="monthly"&post_type="post"&excerpt_length=5&post_html="<li>{title} <div class=\'latest-text\'>{summary}</div></li>"' );

	// array version
	$args = array(
		"header" => "args",
		"stats_views" => 0,
		"limit" => 5,
		"range" => "monthly",
		"post_type" => "post",
		"excerpt_length" => 5,
		"post_html" => "<li>{title} <div class=\'latest-text\'>{summary}</div></li>"
	);
	wpp_get_mostpopular( $args );
	
}
?>
```

Here's the output: http://oi66.tinypic.com/34e3jh3.jpg. As you can see, the _excerpt_length_ parameter worked just fine in both cases.

Are you embedding the wpp_get_mostpopular() template tag in a widget or something similar?

**Comment by gnesher** [comment]
Thanks for the quick reply.

No I've embedded it directly into index.php (it's in the sidebar, but the site is built a little funky and 
I don't use the same sidebar code).

Running the array version gives the same result (you can see it - http://client.gnesher.com/?page_id=42)

I don't usually use Wordpress / index.php just helping a friend so not sure how to debug this. Happy to send a copy of the file?

Thanks

**Comment by cabrerahector** [comment]
Don't know for sure since I'm unable to reproduce it here - although I did test using a different version of the plugin. I'll take another look and get back to you as soon as I can (I'll be going out shortly and won't be back until tomorrow).

If I'm still unable to reproduce it after checking with the same version (you guys are using v.3.3.3, I checked with a much older one by chance), then I might need admin access to the site to debug the code. I'm not sure if your friend would be OK with that, so you might want to ask in advance.
